deploy.sh

```diff
@@ -1,17 +1,21 @@
 #!/bin/bash
 
-# Gets the current directory
-SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
 
-# Create Application Load Balancer for the service 
+# Create Application Load Balancer for the service
 terraform apply
 
 # Apply LB variables exported by terraform
```

Comment:
Maybe add some more detailed comments on how this works. Without reading through all the terraform it would be difficult to know where the vars are coming from.